### PR TITLE
community/pdns: security upgrade to 4.1.5

### PR DESCRIPTION
**Pull request by J0WI**
https://blog.powerdns.com/2018/11/06/powerdns-authoritative-server-4-0-6-4-1-5-and-recursor-4-0-9-4-1-5-released/

### CODE REVIEW

**Comment by J0WI** [comment]
See also:
#5593
#5594
#5595
#5596
#5597
#5598
#5599
#5600
#5601

File: community/pdns/APKBUILD [hunk 1]
```diff
@@ -36,12 +36,14 @@ subpackages="$pkgname-doc $pkgname-openrc $pkgname-tools
...
 source="https://downloads.powerdns.com/releases/$pkgname-$pkgver.tar.bz2
-	libressl-2.7.patch
 	$pkgname.initd
 	$pkgname.conf"
 builddir="$srcdir/$pkgname-$pkgver"
 
 # secfixes:
+#   4.1.5-r0:
+#     - CVE-2018-10851
+#     - CVE-2018-14626
```
Comment (andypost) [inline community/pdns/APKBUILD]: would be great to add this top commit message

Comment (J0WI) [reply community/pdns/APKBUILD]: It's not clear to me what I'm requested to do. The top comment has already been edited, is there anything preventing this from getting merged?

Comment (J0WI) [reply community/pdns/APKBUILD]: @andypost are the CVE numbers required in the commit message to get this merged?
I would have to change all pending 10 PRs...

Comment (andypost) [reply community/pdns/APKBUILD]: IIRC recommented

Comment (J0WI) [reply community/pdns/APKBUILD]: Okay, I've added them.

**Comment by J0WI** [comment]
@ncopa it seems that you committed this and #5593 to the master branch, but all stable branches are still vulnerable. Could you have a look at the provided patches in https://github.com/alpinelinux/aports/pull/5592#issuecomment-437447733?

**Comment by J0WI** [comment]
Fixed in 943fe828eb474fd0c86ec357c79b053b6b7c469a

**Comment by J0WI** [comment]
@ncopa @tcely not all patches have been backported (e.g. `pdns` is still vulnerable in `3.8.2`). Please have a look at the remaining patches:
#5594
#5595
#5596
#5599
#5600
#5601

**Comment by J0WI** [comment]
@ncopa @tcely please have a look at this before the 3.9 release.

**Pull request closed without merging**